pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -306,12 +331,18 @@ func (config *V2RegistriesConf) postProcessRegistries() error {
 		}
 
 		if reg.Prefix == "" {
+			if reg.Location == "" {
+				return &InvalidRegistries{s: "invalid condition: both location and prefix are unset"}
+			}
 			reg.Prefix = reg.Location
 		} else {
 			reg.Prefix, err = parseLocation(reg.Prefix)
 			if err != nil {
 				return err
 			}
+			if reg.Prefix[:2] != "*." && reg.Location == "" {
```

Comment:
This is consistent with the current documentation and fine for now.

For future fix-ups:
- I don’t think `*.` being a special case helps; allowing configuration authors to always use Prefix would simplify them. Compare https://github.com/openshift/runtime-utils/pull/11 .

Reply:
noted as FIXME.